src/jit/lsra.cpp

```diff
@@ -1868,6 +1868,14 @@ void LinearScan::identifyCandidates()
             continue;
         }
 
+#if FEATURE_MULTIREG_ARGS_OR_RET        
+        if (varDsc->lvIsMultiRegArgOrRet)
+        {
+            varDsc->lvLRACandidate = 0;
+            continue;
```

Comment:
In importer we only set `lvDoNotEnregister` in one case (and there are several other return cases that do not). However, I realize now that we don't want to set it for args (only returns), so I think the better near-term fix is to set `lvDoNotEnregister` in `Lowering::treeNodeInfoInit()` so that we don't pessimize the arg case.